connector/connect/common/src/test/resources/query-tests/explain-results/createTable_with_schema.explain

```diff
@@ -0,0 +1,2 @@
+SubqueryAlias spark_catalog.default.test_parquet_table
```

Comment:
Do we need such the way to test `catalog.createTable` API?

Reply:
If not, let me remove this test case

Reply:
removed